pkg/kubectl/cmd/clusterinfo.go

```diff
@@ -82,7 +83,7 @@ func RunClusterInfo(factory *cmdutil.Factory, out io.Writer, cmd *cobra.Command)
 					link += "http://" + ip + ":" + strconv.Itoa(port.Port) + " "
 				}
 			} else {
-				link = client.Host + "/api/v1beta3/proxy/namespaces/" + service.ObjectMeta.Namespace + "/services/" + service.ObjectMeta.Name
+				link = client.Host + "/api/" + latest.Version + "/proxy/namespaces/" + service.ObjectMeta.Namespace + "/services/" + service.ObjectMeta.Name
```

Comment:
The ideal thing to do will be to create a library for generating the appropriate URL (similar to testapi.ResourcePath) and use that here and in https://github.com/GoogleCloudPlatform/kubernetes/blob/master/pkg/client/request.go#L491.

For now, this LGTM. I expect that we will update the API version more frequently then updating the schema.